gcp/extract/lib/results.py

```diff
@@ -124,7 +124,7 @@ def sum_into_data(root, basenames, columns, config, transforms, vectransforms):
                             data[filestuff][rowstuff][(batch, gcm, iam)] += values
                         observations += 1
                         continue
-                
+                    years_for_return = years                
```

Comment:
This isn't used. Please remove.

Reply:
done. sorry this was leftover from an earlier hack I tried and clearly forgot about.